From 37b9c0a8b8f48369a0037c29d0c95f6fb0bcf5ef Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:01:30 -0400
Subject: [usb] core: usb_amd_resume_quirk() can be static

Message-id: <1410887463-20674-19-git-send-email-dzickus@redhat.com>
Patchwork-id: 95105
O-Subject: [RHEL7 PATCH 018/591] usb: core: usb_amd_resume_quirk() can be static
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 00d5f289b26a50e75c5e0ac13d842885fc4c6c8c
Author: Fengguang Wu <fengguang.wu@intel.com>
Date:   Thu Sep 26 11:56:50 2013 -0700

    usb: core: usb_amd_resume_quirk() can be static

    Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
    Acked-by: Huang Rui <ray.huang@amd.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/quirks.c b/drivers/usb/core/quirks.c
index aeda2ee..12924db 100644
--- a/drivers/usb/core/quirks.c
+++ b/drivers/usb/core/quirks.c
@@ -203,7 +203,7 @@ static bool usb_match_any_interface(struct usb_device *udev,
  return false;
 }
 
-int usb_amd_resume_quirk(struct usb_device *udev)
+static int usb_amd_resume_quirk(struct usb_device *udev)
 {
  struct usb_hcd *hcd;
 
-- 
1.7.1